### PR TITLE
v1.4.0 - MotoROS Driver

### PR DESCRIPTION
**Pull request by ted-miller**
Changed the order of the Setup Validation alarms so that the most critical alarms are raised first.

### CODE REVIEW

**Comment by gavanderhoorn** [comment]
Rather difficult to review, seeing as there are no source changes :)

I assume you have tested this on the affected controller(s)?

**Comment by ted-miller** [comment]
Yes.  The only change was a copy/paste to change the order that the parameters.

**Review by gavanderhoorn — APPROVED** [review]

**Comment by gavanderhoorn** [comment]
Ok. I've approved it based on your comment.

Could I ask you to rebase this PR on-top of current `jade-devel`? That would avoid creating unnecessary merge commits.

**Comment by ted-miller** [comment]
I just clicked the merge button.  Is that sufficient?

**Comment by gavanderhoorn** [comment]
That also works, but that creates merge commits, which aren't really necessary.

File: motoman_driver/MotoPlus/CtrlGroup.c
Comment (gavanderhoorn) [inline motoman_driver/MotoPlus/CtrlGroup.c]: `strlen('SLEURBTE') == 8`, not `7`? I think there's an `E` too many in there somewhere.

Comment (ted-miller) [reply motoman_driver/MotoPlus/CtrlGroup.c]: Yes, I will fix that.  However, I did not intend for this commit to be added to the PR yet.  (Another github mistake on my part.)

I'm working with someone to get a Delta robot working.  I have asked him to test this code before I try to push to the main branch.  Right now, these changes are untested.

**Comment by shaun-edwards** [comment]
It looks like I sat on this too long and some other commits got included with this PR.  My bad.  @ted-miller, am I correct in assuming that this [diff](https://github.com/ros-industrial/motoman/pull/148/files/8be33b899732e55ae881ebdf06fba38f4b3d6eae) is the one you want to submit for version: `1.3.9`?

As an aside, any commits to the PR branch (in this case, `jade-devel`), also get added to PR.  I can fix this, as long as you verify the diff above is correct.

**Comment by ted-miller** [comment]
@shaun-edwards  Yes, that diff is correct for v1.3.9.

But... at this point I'm ok with just waiting for testing to complete on v1.4.0.  So, if you don't mind keeping this PR open for a little bit, we'll just skip v139.  I'll keep committing to the same branch if any changes are needed for v140.

**Comment by shaun-edwards** [comment]
@ted-miller, sounds good to me.  It's my intention to approve PRs within a week if not sooner.  If I forget again, please don't hesitate to ping me.  Odds are I missed the PR.

**Comment by ted-miller** [comment]
@shaun-edwards  This update should be good now.  Fraunhofer reports it works on a delta and I have retested on a standard arm.  This update should also allow it work for scara and palletizing robots.

**Comment by gavanderhoorn** [comment]
We should really try to avoid pushing changes for one driver rev to the same branch as a new driver rev; it makes reviews meaningless (the PR status overview shows that I 'approved these changes', but that only really applies to the first commit with the `1.3.9` update).

**Comment by gavanderhoorn** [comment]
Nevertheless, we'll merge this PR now.

I'll also sync `indigo-devel`.

**Pull request merged**
